components/icons/PrivateInfoIcon.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Lock } from '@styled-icons/fa-solid';
+import StyledTooltip from '../StyledTooltip';
+import { useIntl, defineMessages } from 'react-intl';
+
+const msg = defineMessages({
+  defaultContent: {
+    id: 'Tooltip.PrivateInfo',
+    defaultMessage: 'This info is private',
+  },
+});
+
+/**
+ * A lock icon with a tooltip indicating that this info is private
+ */
+const PrivateInfoIcon = ({ children, size, ...props }) => {
+  const { formatMessage } = useIntl();
+  return (
+    <StyledTooltip childrenContainer="span" content={() => children || formatMessage(msg.defaultContent)}>
+      <Lock size={size} {...props} />
```

Comment:
I know this is related to design, but someone raised and I think it would also look cool with `EyeSlash`.

Reply:
Choosing icons is always hard because they vary a lot from one culture/personal experience to another. I'm personally fine with the lock, Facebook use the same one to control the privacy of a post so I don't think it's a bad choice.

What I will always push for is to have a proper label next to the icon. A label always makes things clearer, even if from a design perspective it's annoying because it takes place.

We have a tooltip here to make it explicit, which in my opinion is not perfect because tooltips are not very accessible (for example on mobile).

Reply:
+1 to follow Facebook.